src/nyc_trees/apps/event/templates/event/partials/event_popup.html

```diff
@@ -0,0 +1,13 @@
+<a href="{{ event.get_absolute_url }}" class="event block item rightarrow">
+    <div class="row">
+        <div class="event-date col-xs-3">
+            <div class="event-month">{{ event.begins_at.date|date:"M" }}</div>
+            <div class="event-day h4">{{ event.begins_at.date|date:"d" }}</div>
+        </div>
+        <div class="event-details col-xs-6">
+            <h5 class="event-detail event-detail--title">{{ event.title }}</h5>
+            <div class="event-detail">{{ event.group.name }}</div>
```

Comment:
@jfrankl Is there any other info we need on the action bar here?  (page 93 of the wireframes)